api/user.go

```diff
@@ -58,9 +58,16 @@ func GetUsers(c *gin.Context) {
 		})
 	} else if id == 0 {
 		c.JSON(http.StatusOK, users)
-	} else if len(users) == 1 {
-		c.JSON(http.StatusOK, users[0])
-	} else if len(users) > 1 {
+	}
+	var user []auth.User
```

Comment:
Why do we need this extra filtering here?  The `err = db.Model(organization).Related(&users, "Users")` above should filter by user id if `id` is provided.

Reply:
thx, fixed